### PR TITLE
Redis - pipelined deprecation fixes 

### PR DESCRIPTION
**Pull request by ruban-thilak**
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.

```ruby
redis.multi do
  redis.get("key")
end
```

should be replaced by

```ruby
redis.multi do |pipeline|
  pipeline.get("key")
end
```
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.

```ruby
redis.pipelined do
  redis.get("key")
end
```

should be replaced by

```ruby
redis.pipelined do |pipeline|
  pipeline.get("key")
end
```

### CODE REVIEW

**Comment by darkleaf** [comment]
@ruban-thilak Thank you for your contribution, but I cannot accept it now.
First, we have planned to add CI. I hope this task will be done soon.
Second, please split this PR into two parts and add more info and links.

**Comment by ruban-thilak** [comment]
Sure @darkleaf, I will split this PR. 
If adding CI is the priority, I can set up Rubocop & test in GitHub actions.

File: lib/lowkiq/queue/actions.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def initialize(queue, queries)
       def perform_all_jobs_now
         @pool.with do |redis|
           uredis = Utils::Redis.new redis
-          redis.multi do
+          redis.multi do |pipeline|
```
Comment (darkleaf) [inline lib/lowkiq/queue/actions.rb]: @ruban-thilak please check this point carefully again

**Review by darkleaf — CHANGES_REQUESTED** [review]
.

**Pull request closed without merging**